dask_ml/decomposition/pca.py

```diff
@@ -175,13 +197,16 @@ class PCA(sklearn.decomposition.PCA):
       ``dask.linalg.svd_compressed``.
     * n_components : ``n_components='mle'`` is not allowed.
       Fractional ``n_components`` between 0 and 1 is not allowed.
+    * center : defaults to ``True`` and enables control over whether centering
+      gets implicitly performed as part of the PCA model steps.
```

Comment:
```suggestion
    * center : if ``True`` (the default), automatically center input data before performing PCA.
          Set this parameter to ``False`` if the input data have already been centered before running ``fit()``.
```

I don't think this language is quite correct. This parameter isn't about whether centering is done "implicitly" or "explicitly", but about whether it's done automatically or this class assumes that it's being given already-centered today. Please consider my suggested change to the language.